### DISCUSSION TITLE
Can we give custom retry_delay backoff function? Or a list of retry_delay values?

### DISCUSSION THREAD

**Discussion by workingDice** [apache/airflow#37980] [Q&A] [answered]
For example, if we want our retry delays be: [15min, 1h, 12h, 24h, 48h], is there any easy way to achieve that? ( I didn't find any : ( )

**Comment by potiuk** [comment, marked as answer]
No. But contributions are welcome to add such feature.

**Reply by workingDice** [reply]
thank you for your quick reply
